### DISCUSSION TITLE
Next release, major release?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#2557] [Releases]
If the next release would be a major, I'd like to see these PR merged

- add openmage contributors to copyright https://github.com/OpenMage/magento-lts/pull/2295
- bump minimum php to 7.3 https://github.com/OpenMage/magento-lts/pull/2413
- add ext-intl to requirements https://github.com/OpenMage/magento-lts/pull/2437
- GUI to edit EAV attributes (which is a major feature) https://github.com/OpenMage/magento-lts/pull/2317 https://github.com/OpenMage/magento-lts/pull/2352 https://github.com/OpenMage/magento-lts/pull/2353
- remove more IE code https://github.com/OpenMage/magento-lts/pull/2427
- update extjs https://github.com/OpenMage/magento-lts/pull/2473
- remove legacy media uploader code https://github.com/OpenMage/magento-lts/pull/2434

plus the enhancements we usually add daily.
I'd be happy to see a release like that, also if we don't go to a full composer build yet.

**Comment by elidrissidev** [comment]
We can add to the list a PR which I'll create after #2347 to remove Form Key configs for Contacts, Newsletter, and Checkout, in favor of the global `system/csrf/use_form_key` :).

**Reply by fballiano** [reply]
agree 100%
